nbconvert/exporters/exporter.py

```diff
@@ -316,7 +316,7 @@ def _preprocess(self, nb, resources):
         # to each preprocessor
         for preprocessor in self._preprocessors:
             nbc, resc = preprocessor(nbc, resc)
-            try: 
+            try:
```

Comment:
my text editor Atom must do something with spaces. i hope this isn't problematic. if it is i can try to fix it.